frontend/src/pages/Home.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+	<VContent>
```

Comment:
ensure that your IDE has appropriate config, all this file has tabs instead of spaces